warhorn/warhorn.py

```diff
@@ -158,6 +158,8 @@ def overwrite_files(path, destination, overwrite, logfile, print_log_name):
 
     """
     sub_files, sub_folders = get_subfiles(path)
+    sub_files = filter(lambda f: not f.startswith('.'), sub_files)
```

Comment:
What is the reason for this to be here? If it is just trying to avoid . and ..
it may miss some hidden folders/files that starts with . also

Reply:
Is there a usecase for hidden files/folders to be copied? Here we are trying to avoid hidden files like git etc., from being copied.

Reply:
Thank you for the reply, I was trying to figure why the code should be here, can you add an inline comment stating that these 2 lines are to avoid hidden files?
Right now we don't have any hidden files but we may need to be aware of these lines in the future if we introduce them

Reply:
Thank you for the reply, I was trying to figure why the code should be here, can you add an inline comment stating that these 2 lines are to avoid hidden files?
Right now we don't have any hidden files but we may need to be aware of these lines in the future if we introduce them

Reply:
Done. Please check and approve.